locale/Makefile

```diff
@@ -31,32 +32,28 @@ all-mo: $(MOFILES)
 check: $(POXFILES)
 	msgfmt -c ${POTFILE}
 
-# Merge PO files
-update-po:
-	for f in $(shell find ./ -name "*.po") ; do \
-		msgmerge -N --backup=none -U $$f ${POTFILE} ; \
-	done
-
 # Unify duplicate translations
 uniq-po:
 	for f in $(shell find ./ -name "*.po") ; do \
 		msguniq $$f -o $$f ; \
 	done
 
-tx-pull:
+tx-pull: $(EDITFILES)
 	tx pull -f
-	-git commit -a -m "i18n - extracting new, updating rails, pulling from tx"
```

Comment:
It was intentional.  The old way used to result in one commit as it would amend this original commit later.  It used to do this early commit because it would then revert .po file changes *back*.